tools/gh_api.py

```diff
@@ -130,7 +130,7 @@ def get_paged_request(url, headers=None, **params):
             params = None
             print("fetching %s" % url, file=sys.stderr)
```

Comment:
Missed this one too.
```suggestion
            print(f"fetching {url}", file=sys.stderr)
```